src/parser.ts

```diff
@@ -3223,7 +3223,7 @@ export class Parser {
     parseExportSpecifier(): Node.ExportSpecifier {
         const node = this.createNode();
 
-        const local = this.matchKeyword('default') ? this.parseNonComputedProperty() : this.parseVariableIdentifier();
+        const local = this.parseNonComputedProperty();
```

Comment:
The name is still kind of confusing. Expect a future refactoring that changes it to `parseIdentifierName`.